src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/ProjectImports/ProjectImportsSubTreeProvider.ImplicitProjectCheck.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Tree.ProjectImports
+{
+    internal sealed partial class ProjectImportsSubTreeProvider
+    {
+        private sealed class ImplicitProjectCheck
+        {
+            private readonly string _programFiles86;
+            private readonly string? _programFiles64;
+            private readonly string _windows;
+
+            public string? ProjectExtensionsPath { get; set; }
+
+            public ImplicitProjectCheck()
+            {
+                _windows        = Environment.GetFolderPath(Environment.SpecialFolder.Windows);
+                _programFiles86 = Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86);
+
```

Comment:
Does there need to be an entry here for any paths coming from the Visual Studio installation?

(i'll try to get this branch running locally so I'll be able to answer this myself :) )

Reply:
Yes, we do. Microsoft.Common.props and others all show up as editable for me, because it comes from E:\VSInstalls\Master\MSBuild\Current\Microsoft.Common.props on my machine.
![image](https://user-images.githubusercontent.com/754264/69775890-643f9a80-11ee-11ea-9fb1-6a3f17b5d749.png)

Reply:
Thank you, I'll be sure to handle this. The `VSAPPIDDIR` environment variable might do the job.

Reply:
Fixed.

![image](https://user-images.githubusercontent.com/350947/70029728-d0117100-15fb-11ea-8ac4-238bb0da1631.png)